crates/blenvy/src/components/process_gltfs.rs

```diff
@@ -145,10 +145,11 @@ pub fn add_components_from_gltf_extras(world: &mut World) {
 
             {
                 let mut entity_mut = world.entity_mut(entity);
-                type_registration
-                    .data::<ReflectComponent>()
-                    .expect("Unable to reflect component")
-                    .insert(&mut entity_mut, &*component, &type_registry);
+                let Some(reflected_component) = type_registration.data::<ReflectComponent>() else {
+                    warn!(?component, "unable to reflect component");
+                    continue;
+                };
+                reflected_component.insert(&mut entity_mut, &*component, &type_registry);
 
                 entity_mut.insert(GltfProcessed); //
```

Comment:
```suggestion
                entity_mut.insert(GltfProcessed);
                
                let Some(reflected_component) = type_registration.data::<ReflectComponent>() else {
                    warn!(?component, "unable to reflect component");
                    continue;
                };
                reflected_component.insert(&mut entity_mut, &*component, &type_registry);
```
It seems like it is done processing even when the reflection failed, isn't it?

Reply:
ah true, fixing